docs/docs/workspaces/sso/README.md

```diff
@@ -0,0 +1,67 @@
+# Single Sign-On Overview
+
+Pipedream supports Single Sign-On (SSO) using SAML or Google OAuth, which allows IT and team administrators easier controls to manage access and security.
+
+Using SSO with your Identity Provider (IdP) centralizes user login management and provides a single point of control for IT teams and employees.
+
+[[toc]]
+
+## Requirements for SSO
+- Your workspace must be on a [Business or Enterprise plan](https://pipedream.com/pricing)
+- If using SAML, your Identity Provider must support SAML 2.0
+- Only workspace admins and owners can configure SSO
+- Your workspace admin or owner must verify ownership of the SSO email domain
+
+:::tip The below content is for workspace admins and owners
+Only workspace admins and owners have access to add verified domains, set up SSO, and configure workspace login methods
+:::
+
+## Verifying your Email Domain
+In order to configure SAML SSO for your workspace, you first need to verify ownership of the email domain. If configuring Google OAuth (not SAML), you can skip this section.
+
+1. Navigate to the [Verified Domains](https://pipedream.com/settings/domains) section of your workspace settings
+2. Enter the domain you'd like to use for SSO (for example, `your-company.com`), then click **Add Domain**
```

Comment:
We use `example.com` in all example cases in the docs because it's reserved for that purpose via RFC.